getting_started_tutorials/cuml_sklearn_colab_demo.ipynb

```diff
@@ -0,0 +1,4310 @@
+{
```

Comment:
<p>We went from 45 seconds to 2.5 seconds on a 20K row dataset with 100 columns. We should call this out! People will be able to immediately understand how impactful that will be if they've got 50K, 100K, 1M etc. rows.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/showcase/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>